tests/integration/test_join.py

```diff
@@ -121,6 +121,9 @@ def test_join_cross(c, user_table_1, department_table):
     assert_eq(return_df, expected_df, check_index=False)
 
 
+@pytest.mark.skip(
```

Comment:
While some of the optimization work in ongoing, we can maybe create an issue for these cases where the optimized plan isn't handled where the issue can track the optimized and non optimized version of the plans.